internal/services/network/network_interface_resource_test.go

```diff
@@ -388,6 +416,62 @@ resource "azurerm_network_interface" "test" {
 `, r.template(data), data.RandomInteger)
 }
 
+func (r NetworkInterfaceResource) auxiliaryNone(data acceptance.TestData) string {
+	// Auxiliary Mode Nic is enabled in specific regions (https://learn.microsoft.com/en-us/azure/networking/nva-accelerated-connections#supported-regions) for now
+	// To not affect other testcases of `Network`, hard-code to that for now
+	data.Locations.Primary = "westus"
+
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_network_interface" "test" {
+  name                          = "acctestni-%d"
+  location                      = "%s"
+  resource_group_name           = azurerm_resource_group.test.name
+  enable_accelerated_networking = true
+
+  ip_configuration {
+    name                          = "primary"
+    subnet_id                     = azurerm_subnet.test.id
+    private_ip_address_allocation = "Dynamic"
+  }
+
+  tags = {
+    fastpathenabled = "true"
```

Comment:
what's this?

Reply:
It's a custom tag which is mentioned in https://learn.microsoft.com/en-us/azure/networking/nva-accelerated-connections#prerequisites and is used to enable `AuxiliaryMode`. Users will get detailed guidance after signing up for this feature mentioned in https://learn.microsoft.com/en-us/azure/networking/nva-accelerated-connections#prerequisites